src/User32.Desktop/User32.cs

```diff
@@ -647,13 +1103,71 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
         public static extern int RegisterWindowMessage(string lpString);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool GetWindowDisplayAffinity(IntPtr hWnd, out int dwAffinity);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool EnumDisplayMonitors(IntPtr hdc, IntPtr lprcClip, MonitorEnumDelegate lpfnEnum, IntPtr dwData);
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool GetClassInfoEx(IntPtr hInstance, string lpClassName, ref WNDCLASSEX lpWndClass);
```

Comment:
The `out` parameter I got, but didn't I just heard before on this PR that the ` [In] char*`  would preferably be mapped as `string` ?

When should I map the ` LPTSTR`  as `string` and when should it be `[Friendly(In|Array)] char*` ?